config/config.py

```diff
@@ -1,6 +1,23 @@
+from random import randint
 class Config(object):
     price_range = [0, 10]  # say for a uniform distribution
     quantity_range = [100, 1500]  # again uniform?
-    region = [[0, 200], [0, 200]]
     run_length = 250
     random_agent_creation = .05
+    number_of_subregions = 9
+    subregionx = 100
+    subregiony = 100
+    region = [] #Where there are 9 subregions, numbering from 0 to 8
```

Comment:
Do _not_ initialise anything difficult in the `Config` object. It should be just data, nothing else.

All other initialisation of region coordinates should follow through `run.py`.